test/fw/ptl/lib/eatcpu.pl

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/perl -w
+
+# Usage: eatcpu.pl [ seconds [units] ]
+# Eats CPU resources for time specified by user
+# Handles TSTP and CONT signals
+# Limitations:
+# 1.SIGSTOP should be changed to TSTP during MOM configuration
+#   $suspensig TSTP_number
```

Comment:
ptl/lib is the wrong place for this file.  This is a testing file, not an interface file.  At best it should be in ptl/utils.  I don't think it needs to be checked in at all.  I think you can create a much smaller piece of python code and embed it into your test script itself.  Write it out using create_temp_file() and submit it as the job.